pkg/iac/deserializers.go

```diff
@@ -26,6 +26,7 @@ func Deserializers() []deserializer.CTYDeserializer {
 		awsdeserializer.NewLambdaFunctionDeserializer(),
 		awsdeserializer.NewDBSubnetGroupDeserializer(),
 		awsdeserializer.NewVPCSecurityGroupDeserializer(),
+		awsdeserializer.NewDefaultSecurityGroupDeserializer(),
```

Comment:
Maybe we should name it `DefaultVPCSecurityGroupDeserializer` to be more consistent in term of naming with `VPCSecurityGroupDeserializer`

Reply:
It could be a good discussion to have it normalized indeed. We need to make it a way so that new contributor must not have to think about this kind of naming for a deserializer and a supplier. Let me explain.

On terraform the resource is aws_default_security_group and aws_security_group. At first, if I remembered well, I decided to name it so that the template would be `service_resource`, thus `vpc_security_group` or `ec2_ami` or `vpc_subnet`. But as far as we are now, it is more something like juste `resource`, thus it would be `security_group` or `subnet`.

We need to get it over with now so that we can change past resource with wrong naming.

Reply:
Hmm let's move the topic to another PR, we need to rethink resources / suppliers and deserializer naming to avoid collisions.